.coveragerc

```diff
@@ -228,6 +228,7 @@ omit =
     homeassistant/components/light/lifx_legacy.py
     homeassistant/components/light/limitlessled.py
     homeassistant/components/light/osramlightify.py
+    homeassistant/components/light/pwm.py
```

Comment:
I think that the name is too generic. Prefixing with `rpi_gpio` would help to identify the it as most people will use this platform with a Raspberry Pi.